fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleType.java

```diff
@@ -101,6 +101,7 @@ public enum RuleType {
     EXTRACT_AND_NORMALIZE_WINDOW_EXPRESSIONS(RuleTypeClass.REWRITE),
     CHECK_AND_STANDARDIZE_WINDOW_FUNCTION_AND_FRAME(RuleTypeClass.REWRITE),
     CHECK_MATCH_EXPRESSION(RuleTypeClass.REWRITE),
+    CREATE_PTOPN_FOR_WINDOWN(RuleTypeClass.REWRITE),
```

Comment:
Avoid abbreviations whenever possible, as they are difficult to understand
```suggestion
    CREATE_PARTITION_TOPN_FOR_WINDOW(RuleTypeClass.REWRITE),
```

Reply:
done